spec/src/main/asciidoc/ch12-xml-or-mapping-descriptor.adoc

```diff
@@ -38,12 +38,10 @@ subelement is not specified, the XML descriptor overrides the values set
 or defaulted by the use of annotations, as described below.
 
 The mapping files used by the application
-developer must conform to the XML schema defined in <<a17175>> or to the object/relational
-mapping schema defined in a previous version of this specification
-<<a19493>>, <<a19501>>.
+developer must conform to the XML schema defined in <<a17175>>.
 
-The Java Persistence persistence provider
-must support use of older versions of the object/relational mapping
+The Jakarta Persistence persistence provider
+may support use of older versions of the object/relational mapping
 schema as well as the object/relational mapping schema defined in
```

Comment:
I think that the orm_2_2.xsd can also be included as per https://eclipse-ee4j.github.io/jakartaee-platform/minutes/2020-01-28.html